src/components/Tree/Tree.stories.tsx

```diff
@@ -327,3 +333,50 @@ export const WithExpandOnSelect = ({ ...args }: TreeProps) => {
         </div>
     );
 };
+
+export const InsideInlineDialog = ({ ...args }: TreeProps) => {
+    const [expandedIds, setExpandedIds] = useState<string[]>([]);
+
+    const handleItemExpand = useCallback((id: string) => {
+        setExpandedIds((ids) => [...ids, id]);
+    }, []);
+
+    const handleItemShrink = useCallback((id: string) => {
+        setExpandedIds((ids) => ids.filter((itemId: string) => itemId !== id));
+    }, []);
+
+    const [isOpen, setIsOpen] = useState(false);
+
+    return (
+        <Container maxWidth={'800px'}>
+            <InlineDialog
+                open={isOpen}
+                modality={Modality.NonModal}
+                enablePortal={true}
+                handleClose={() => setIsOpen(false)}
+            >
+                <InlineDialog.Trigger>
+                    <Button
+                        style={ButtonStyle.Default}
+                        emphasis={ButtonEmphasis.Strong}
+                        onClick={() => setIsOpen(!isOpen)}
+                    >
+                        Click me
+                    </Button>
+                </InlineDialog.Trigger>
+                <InlineDialog.Content>
+                    <DialogBody>
+                        <TreeView
+                            {...args}
+                            expandedIds={expandedIds}
+                            onExpand={handleItemExpand}
+                            onShrink={handleItemShrink}
+                        >
+                            {treeItemsMock.map(renderTreeItemLabel)}
+                        </TreeView>
+                    </DialogBody>{' '}
```

Comment:
what's the {'  '} for?

Reply:
nothing... just missed it when cleaning :)  good catch!
cleaned now!